de.dlr.sc.virsat.model/model/dvlm.genmodel

```diff
@@ -37,7 +38,8 @@
         <genFeatures property="None" children="true" createChild="true" ecoreFeature="ecore:EReference dvlm.ecore#//structural/StructuralElementInstance/relationInstances"/>
         <genFeatures property="None" children="true" createChild="true" ecoreFeature="ecore:EReference dvlm.ecore#//structural/StructuralElementInstance/children"/>
         <genFeatures property="None" notify="false" createChild="false" ecoreFeature="ecore:EReference dvlm.ecore#//structural/StructuralElementInstance/parent"/>
-        <genOperations ecoreOperation="dvlm.ecore#//structural/StructuralElementInstance/getDeepChildren"/>
+        <genOperations ecoreOperation="dvlm.ecore#//structural/StructuralElementInstance/getDeepChildren"

```

Comment:
What are these changes about? They seem to be connected to the missing annotations, aren't they.